### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by AlbertoElias**
So there are currently two issues.

1. Textrequireify has been made into a custom loader, but I need to get the path of the file that's being modified and that's not working. I asked in [stack overflow](http://stackoverflow.com/questions/32320038/cant-access-resourcepath-property-from-webpack-loader) (following the webpack contributing guidelines) to see if I'm doing something wrong. The other alternative is to use a transform loader like the web app team and just use the original code, but we need to pass in the cwd as a config option, and the transform loader doesn't support passing in custom parameters. Check out [this issue](https://github.com/webpack/transform-loader/pull/8)

2. Webpack doesn't act the same way as browserify and still builds when it comes upon certain errors like unmet dependencies. Need to figure out a way around this. @samgiles check out [this](https://github.com/Financial-Times/origami-build-tools/compare/eslint...webpack?expand=1#diff-5a41c7b9bce0c7c00e0f564f21cb7603R92)

### CODE REVIEW

**Comment by antoligy** [comment]
Unfortunately this beta currently breaks our build process as bower-webpack-plugin does not respect the directory setting in .bowerrc.

I've created lpiepiora/bower-webpack-plugin#25 addressing this, and will attempt to contribute a PR addressing that. (If that gets declined, or isn't merged in time, happy to contribute similar here.)

**Comment by antoligy** [comment]
Submitted piepiora/bower-webpack-plugin#27.

**Comment by antoligy** [comment]
Submitted lpiepiora/bower-webpack-plugin#27.

**Comment by antoligy** [comment]
Unfortunately with that applied there are still some failures, due to the hardcoded `bower_components` in the OBT build task.

**Comment by antoligy** [comment]
Should be resolved in #310. :smile:

**Comment by AlbertoElias** [comment]
This is ready to review. @triblondon could you take a look?

File: lib/origami-build-tools-cli.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ var argument = argv._[0];
 
 function reportTaskError(error) {
 	if (Array.isArray(error)) {
-		log.primaryError(error.join('\n'));
+			log.primaryError(error.join('\n'));
```
Comment (triblondon) [inline lib/origami-build-tools-cli.js]: weird indent?

File: lib/origami-build-tools.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 'use strict';
 
-require('es6-promise').polyfill();
```
Comment (triblondon) [inline lib/origami-build-tools.js]: So we now depend on Node 0.12 minimum?  Are we expressing that anywhere?

Comment (AlbertoElias) [inline lib/origami-build-tools.js]: Node v4. In the package.json. But it works fine with Node 0.12 as well.

**Comment by triblondon** [comment]
Did a review with Alberto.  I suggest capturing more of the gotchas into file and line comments, otherwise looks good.

**Pull request merged**